README.md

```diff
@@ -182,13 +182,19 @@ If you for any reason find yourself behind a transparent proxy you need to set t
 - Set the environment variables permanently by adding the above export commands to your `~/.bashrc` or equivalent.
 
 ##### MacOS
-MacOS uses its own keychain, which means you have no certificates-file to point to. You can solve this by exporting all certificates present in the keychain to a file. 
+MacOS uses its own keychain, which means you have no certificates-file to point to. You can solve this by exporting all certificates present in the keychain to a file.
 
-```bash
-security find-certificate -a -p > certs.pem
+```zsh
+security find-certificate -a -p > ~/certs.pem
 ```
 
-The command above will export all system and personal certificates in your keychain to `certs.pem`. You can now follow the first steps, and point your env-variables to `certs.pem`.
+The command above will export all system and personal certificates in your keychain to `certs.pem` in `~` (your user home)
+
+```zsh
+ printf '\n### You added this for vagrant box certs ###\nexport SSL_CERT_FILE=~/certs.pem CURL_CA_BUNDLE=~/certs.pem\n' >> ~/.zshrc
+ source ~/.zshrc
```

Comment:
I'm not sure, but I don't think all Mac users use zsh. You should specify an `bash`-alternative or at least why this won't work if you are using `bash`.

Reply:
https://support.apple.com/en-us/HT208050 <- zsh is the default for some time now, and all company Macs would use zsh. I'll leave it for now.